tests/test.messages.js

```diff
@@ -0,0 +1,16 @@
+import * as messages from 'messages';
+
+describe('Messages', function() {
+
+  it('should only have codes with a length < 25', () => {
```

Comment:
That's a good test to have; prevents `E_TOO_LONG_ERROR_NAMES_THAT_ARE_A_BIT_OVERKILL` too. :+1: